ledger/sandbox-on-x/reference.conf

```diff
@@ -504,6 +504,12 @@ ledger {
         # is needed to prevent excessive memory pressure.
         max-input-buffer-size = 50
 
+        # The parallelism factor for the Package Metadata In-memory view loading pipeline.
+        package-metadata-view-load-parallelism = 16
+
+        # The parallelism factor for the Package Metadata In-memory view process pipeline.
+        package-metadata-view-process-parallelism = 16
```

Comment:
just for my understanding, what does it mean that we have a default here and also in the backing case class as well? Why do we need both?

Reply:
`reference.conf` is just a file with documentation of all the possible options. Values defined here are not being used, but the idea is that if someone is using this file - it should work and have meaningful defaults

Reply:
:+1: